packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -78,10 +80,15 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
   constructor(props: IFocusZoneProps) {
     super(props);
 
-    this._warnDeprecations({
-      rootProps: undefined,
-      allowTabKey: 'handleTabKey'
-    });
+    // Manage componentRef resolution.
+    initializeComponentRef(this);
+
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
Nice, components spend a bit of time in these calls so we'll get some savings in production builds 👍

Reply:
Should we move this check to the inside of `warnDeprecations`? Seems the best place if we want to standardize this behavior.

Reply:
this just further removes the call completely